src/ScriptEditor/ScriptEditor.ts

```diff
@@ -100,6 +103,25 @@ export class ScriptEditor {
           2686,
         ],
       });
+      // Sync all javascript and typescript text models to both language servers.
+      {
+        // `monaco.languages.typescript.get{Java,Type}ScriptWorker` returns a promise that
+        // fires with a function that takes a list of `monaco.Uri`s and sync's them with the
+        // worker. (It also returns the worker, but we don't care about that.)
+        const languageWorker = new Promise<(...uris: monaco.Uri[]) => unknown>((resolve) =>
+          monaco.languages.onLanguage(language, () => getLanguageWorker().then(resolve)),
```

Comment:
Why do you need to jump through these extra callback hoops with onLanguage? Ultimately, `languageWorker` is just `getLanguageWorker()` with extra steps.

Reply:
Because `getLanguageWorker()` errors if the language has not yet been loaded, unfortunately.  I think I had meant to include that in the comment, but it appears that I missed adding that bit.

Reply:
man wtf. It's a flipping promise, it can wait to resolve XD